en/firefox_cloud_services_tos.md

```diff
@@ -18,16 +18,14 @@ This top section summarizes the terms below. This summary is provided to help yo
 
 2. #### Firefox Accounts
 
-    In order to use some of the Services, you'll need to create a Firefox Account.  During registration, you will be asked to set a password. You are responsible for keeping your password confidential and for the activity that happens through your Firefox account. Mozilla is not responsible for any losses arising out of unauthorized use of your Firefox account.
-    
-    “Join Firefox” is an offering available in some countries to better experience the Firefox family of products as well as practical privacy and security knowledge.
+    In order to use some of the Services, you'll need to create a Firefox Account.  During registration, you will be asked to set a password. You are responsible for keeping your password confidential and for the activity that happens through your Firefox Account. Mozilla is not responsible for any losses arising out of unauthorized use of your Firefox account.
```

Comment:
Could we drop the double spaces after punctuation? They're not effective anyway when rendered, and they're used iconsistently.